llumnix/entrypoints/vllm/api_server.py

```diff
@@ -224,34 +225,25 @@ async def is_ready():
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument("--host", type=str, default="localhost")
-    parser.add_argument("--port", type=int, default=8000)
-    parser.add_argument("--ssl-keyfile", type=str, default=None)
-    parser.add_argument("--ssl-certfile", type=str, default=None)
-    parser.add_argument('--disable-log-requests-server',
-                        action='store_true',
-                        help='disable logging requests in server')
-    parser.add_argument("--ray-cluster-port", type=int, default=30050)
-    parser.add_argument('--launch-ray-cluster',
-                        action='store_true',
-                        help='if launch ray cluster in api server')
-
-    parser = EngineManagerArgs.add_cli_args(parser)
+    parser.add_argument("--config-file", default="", metavar="FILE", help="path to config file")
```

Comment:
can we allow to override the parameters using CLI? it's sometimes more convenient to use CLI, for example, when you want to change parameters using a shell script